docs/user_guide/model-registration.rst

```diff
@@ -5,84 +5,60 @@
 Model registration
 ==================
 
-This guide presents a quick instruction for "registering a model" in a project workflow.
+This guide presents instructions for "registering a model" for a project.
 
-It is still recommended to read the detailed instructions under :ref:`model_mapping` and
-:ref:`region` in particular.
-
-Region processing
------------------
-
-"Registering a model" for a **nomenclature**-based project workflow requires two
-specifications: 
+Model registration for a **nomenclature**-based project requires two specifications:
 
+* | a list of region names as they should appear in the processed scenario data
```

Comment:
Is there a reason for the pipe character here and in the following line?

Reply:
This is rst-formatting for enforcing a line break before the example